internal/stage7.go

```diff
@@ -64,7 +70,7 @@ func testType2(stageHarness *test_case_harness.TestCaseHarness) error {
 
 	for _, executable := range nonAvailableExecutables {
 		command := fmt.Sprintf("type %s", executable)
-		actualPath := getPath(executable)
+		actualPath := getPath(executable, exePath)
```

Comment:
This shouldn't use `getPath` at all, misleading naming - no one would expect `getPath` to return `not found`. Let's just create the expected value here directly, and let's also not call it `actualPath`. 

cc: @ryan-gang (no action needed as @glokta1 will fix this, just highlighting naming issues from the past)

Reply:
@rohitpaulk So, should i just remove `getPath` altogether?

Reply:
Yep that could work